### DISCUSSION TITLE
Sülearvutite remont

### DISCUSSION THREAD

**Discussion by LisaWeiss** [ValveSoftware/Dota2-Gameplay#16968] [Ideas]
Öelge mulle, mida teha, kui sülearvutiga on probleeme? Kuhu on kõige parem minna?

**Comment by MaxFolls** [comment]
Enda nimel võin soovitada teil itRemontiga ühendust võtta. See on tõestatud variant. Käisin seal teenindusel [arvutiteeninduskeskus Talin](https://itremont.ee/arvutite-remont/) ja ma ei kahetse seda, seal töötavad suurte kogemustega meistrid, kes aitavad kõigil sellega toime tulla.
